tracer/src/Datadog.Trace/Agent/Transports/HttpStreamRequestFactory.cs

```diff
@@ -29,7 +31,7 @@ public Uri GetEndpoint(string relativePath)
             // https://github.com/grpc/grpc-go/issues/2628 and issue/discussion in aspnetcore here:
             // https://github.com/dotnet/aspnetcore/issues/18522.
             // To play it safe, use localhost as the host instead of the UDS socket name/ named pipe
```

Comment:
Can this comment be updated or moved? The code no longer explicitly uses `http://localhost`

Reply:
Good point, it should probably be added to the constructor/callee 🙂 I'll do that in #2617, just to save hitting the CI Again 🙂